fenix/app/src/main/java/org/mozilla/fenix/debugsettings/tabs/TabTools.kt

```diff
@@ -278,40 +291,25 @@ private fun TabCreationTool(
             text = stringResource(id = R.string.debug_drawer_tab_tools_tab_creation_tool_button_text_private),
             enabled = !hasError,
             onClick = {
-                onAddTabsToPrivateClick(tabQuantityToCreate.toInt())
+                onCreateTabsClick(tabQuantityToCreate.toInt(), false, true)
             },
         )
     }
 }
 
 private data class TabToolsPreviewModel(
-    val activeTabCount: Int = 0,
-    val inactiveTabCount: Int = 0,
-    val privateTabCount: Int = 0,
     val inactiveTabsEnabled: Boolean = true,
-) {
-    val totalTabCount = activeTabCount + inactiveTabCount + privateTabCount
-}
+)
 
 private class TabToolsPreviewParameterProvider : PreviewParameterProvider<TabToolsPreviewModel> {
     override val values: Sequence<TabToolsPreviewModel>
         get() = sequenceOf(
             TabToolsPreviewModel(
-                activeTabCount = 5,
-                inactiveTabCount = 7,
-                privateTabCount = 10,
                 inactiveTabsEnabled = true,
             ),
             TabToolsPreviewModel(
-                activeTabCount = 25,
                 inactiveTabsEnabled = false,
             ),
-            TabToolsPreviewModel(
-                activeTabCount = 50,
-                inactiveTabCount = 700,
-                privateTabCount = 10,
-                inactiveTabsEnabled = true,
-            ),
```

Comment:
these preview permutations weren't adding a ton of value, so I collapsed them to two to just cover inactive tabs being enabled/disabled. We can still in the future add any preview parameters though, now that the scaffolding is in place.